py3-awscrt.yaml

```diff
@@ -38,9 +38,16 @@ pipeline:
     with:
       repository: https://github.com/awslabs/aws-crt-python
       tag: v${{package.version}}
-      expected-commit: e85656611c0172aea7fc990aa16fdf3bdbcc1218
+      expected-commit: 7e52d8a156d44e13afb571b674893441fc94541e
       recurse-submodules: true
 
+  - runs: |
+      # Update s2n to solve compilation error "implicit declaration of function 's2n_cleanup_final'; did you mean 's2n_cleanup_thread'?"
+      # Fix is available in s2n version 1.5.7 https://github.com/awslabs/aws-c-io/issues/688
+      # Bumping to that commit; this should be removable once aws-crt-python bumps the submodule version too.
+      cd crt/s2n
+      git checkout 9f4baecc753d6fe01c13e4f422d2e327c64d06b8
```

Comment:
@hbh7 A post merge thought... 😇 

Will this break the package at some point in the future? Like will this start downgrading a dependency when the main project has moved past this?

Reply:
I think it's entirely possible it might silently downgrade it in the future, yeah. Maybe we want to open an issue and set a reminder to see if it can be removed in maybe in a month or two.